ui/app/util.js

```diff
@@ -212,6 +212,7 @@ function readableDate (ms) {
 }
 
 function isHex (str) {
+  if (str.startsWith('0x')) str = str.replace('0x', '')
```

Comment:
Just to lightly expand, a utility function like this is probably best when it's a [pure function](https://en.wikipedia.org/wiki/Pure_function).

Functional programming has some great benefits, so if you try to compose code functional when possible, you can find it easier to easily chain things together, for example.